packages/web/src/components/Home/FeatureOptions/FindingData/index.ts

```diff
@@ -0,0 +1,28 @@
+import FilterColumn from "./FilterColumn.png";
+import FilterValue from "./FilterValue.png";
+import Group from "./Group.png";
+import Sort from "./Sort.png";
+import { Feature } from "../Feature";
+
+export default {
+    id: 1,
+    text: "Find target files quickly",
+    slides: [
+        {
+            imgSrc: Group,
+            caption: `Unlike traditional file systems, BioFile Finder allows you to dynamically generate your folder structure based on the metadata of your files. In this screenshot, the files are organized by (Grouped by) the "Structure" and "Plate Barcode" columns found in the selected data source.`,
```

Comment:
I'm slightly leaning towards "are organized (grouped) by" or just taking out the capital G in Grouped